test/functionalTest/cases/1112_metadata_as_keymap/metadata_with_dots_v1.test

```diff
@@ -0,0 +1,179 @@
+                    "value": "sweet"
+                }
+            ]
+        }
+        ]
+    }
+    ],
+    "updateAction": "APPEND"
+}'
+orionCurl --url "$url" --payload "${payload}"
+echo
+echo
+
+
+echo "02. Retieve entity"
+echo "=================="
+orionCurl --url "/v1/contextEntities/E"
+echo
+echo
+
+
+echo "03. Check the '=' in DB"
+echo "======================="
+mongoCmd ${CB_DB_NAME} 'db.entities.find({ "_id" : { "id" : "E",  "type": "T", "servicePath" : "/" } }, {_id:0, attrs: 1})'
+echo
+echo
+
+
+--REGEXPECT--
+01. Create entity with metadata with dot in the name
+====================================================
+HTTP/1.1 200 OK
+Content-Length: 685
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "contextResponses": [
+        {
+            "contextElement": {
+                "attributes": [
+                    {
+                        "metadatas": [
+                            {
+                                "name": "m.x",
+                                "type": "string",
+                                "value": "black"
+                            },
+                            {
+                                "name": "m.y",
+                                "type": "string",
+                                "value": "sweet"
+                            }
+                        ],
+                        "name": "A",
+                        "type": "T",
+                        "value": ""
+                    }
+                ],
+                "id": "E",
+                "isPattern": "false",
+                "type": "T"
+            },
+            "statusCode": {
+                "code": "200",
+                "reasonPhrase": "OK"
+            }
+        }
+    ]
+}
+
+
+02. Retieve entity
+==================
+HTTP/1.1 200 OK
+Content-Length: 535
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "contextElement": {
+        "attributes": [
+            {
+                "metadatas": [
+                    {
+                        "name": "m.x",
+                        "type": "string",
+                        "value": "black"
+                    },
+                    {
+                        "name": "m.y",
+                        "type": "string",
+                        "value": "sweet"
+                    }
+                ],
+                "name": "A",
+                "type": "T",
+                "value": "foo"
+            }
+        ],
+        "id": "E",
+        "isPattern": "false",
+        "type": "T"
+    },
+    "statusCode": {
+        "code": "200",
+        "reasonPhrase": "OK"
+    }
+}
+
+
+03. Check the '=' in DB
+=======================
+{ "attrs" : { "A" : { "type" : "T", "creDate" : REGEX(.*), "value" : "foo", "md" : { "m=x" : { "type" : "string", "value" : "black" }, "m=y" : { "type" : "string", "value" : "sweet" } }, "mdNames" : [ "m.x", "m.y" ] } } }
```

Comment:
What is important to review here is that the key in the `md` maps correctly uses `=` as encoding character for dots.